cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1/types.go

```diff
@@ -70,6 +72,20 @@ type AppRepositoryCustomCA struct {
 	SecretKeyRef corev1.SecretKeySelector `json:"secretKeyRef,omitempty"`
 }
 
+// FilterRulesSpec defines a set of rules and aggreagation logic
```

Comment:
FTR: I guess this object will (in the future) be part of a POST HTTP request. If so, we should also model it in the openapi doc. I can take it, it's just for us to remember it.

Reply:
right, I can take a look to that later.

Reply:
See comments on the design doc (sorry for being late on that).

Reply:
adapted to the new state of the document